docs/reference/stable-patch-format.rst

```diff
@@ -0,0 +1,366 @@
+Stable patch format
+===================
+
+Every Ubuntu LTS release during standard security maintenance period (see
+`The Ubuntu lifecycle and release cadence`_) welcomes contributions from
+anyone. However, patches must comply with the required format.
+
+.. _The Ubuntu lifecycle and release cadence: https://ubuntu.com/about/release-cycle
+
+Prerequisites
+-------------
+
+Please subscribe `here
+<https://lists.ubuntu.com/mailman/listinfo/kernel-team>`__ to join
+kernel-team@lists.ubuntu.com before submitting your first patch. Messages
+from non-subscribers will be held in a queue pending admin approval.
+
+Prepareing commits
+------------------
+
+Every patch **must** adhere to the following guideline.
+
+Subject line
+^^^^^^^^^^^^
+
+#. Every patch submitted to a stable kernel **must** have have its subject line
+   starting with "[SRU]" followed by the release name against which the
+   patch is targeted. That release name **must** be enclosed in "[ ]" brackets
+   and **can** be abbreviated using the first letter of the release name (e.g.
```

Comment:
sorry, @lkpdn , i think we are overusing the bold emphasis here.

```suggestion
   and can be abbreviated using the first letter of the release name (e.g.
```

Reply:
Hm, ok. I didn't feel so tbh (ref. https://wiki.ubuntu.com/Kernel/Dev/StablePatchFormat) but will fix. Thanks!